website/docs/jellyfish/api/wallet.md

```diff
@@ -263,7 +314,9 @@ Send given amounts to multiple given address and return a transaction id.
 
 ```ts title="client.wallet.sendMany()"
 interface wallet {
-  async sendMany (amounts: Record<string, number> , subtractfeefrom: string [] = [], options: SendManyOptions = {}): Promise<string>
+  async
+
+  sendMany (amounts: Record<string, number>, subtractfeefrom: string [] = [], options: SendManyOptions = {}): Promise<string>
```

Comment:
what happen on this line break??
the previous one was good

Reply:
Okay. I think they got formatted by IntelliJ I'll revert these changes.